noir-projects/noir-contracts/contracts/private_token_contract/src/types/token_note.nr

```diff
@@ -19,14 +19,14 @@ trait PrivatelyRefundable {
         fee_payer_npk_m_hash: Field,
         sponsored_user_npk_m_hash: Field,
         funded_amount: Field,
-        refund_nonce: Field
-    ) -> [EmbeddedCurvePoint; 2];
+        randomness: Field
+    ) -> (EmbeddedCurvePoint, EmbeddedCurvePoint);
```

Comment:
Returning tuple instead of field allows us to nicely decompose the return value:
` (fpc_point, user_point) = TokenNote::generate_refund_points(...)`